logux/core.py

```diff
@@ -465,7 +465,7 @@ def is_match(cls, channel: str) -> bool:
 
     # Required and optional action methods (these methods should be implemented by consumer)
     @abstractmethod
-    def load(self, action: Action, meta: Meta) -> None:
+    def load(self, action: Action, meta: Meta) -> Action:
```

Comment:
returning just a single action is a rare case. Often it will be an array of actions or array of `[[action, meta]]`, because it is important to set `meta.time`.

Reply:
I think we should merge it as is. Because all of this will be rewritten for proto.v4 support.